.rubocop.yml

```diff
@@ -52,6 +52,9 @@ Metrics/ModuleLength:
 Metrics/BlockLength:
   Enabled: false
 
+Naming/UncommunicativeMethodParamName:
+  Enabled: false
```

Comment:
what the motivation to disable it globally? Is there some specific cases where we can not fix it? If so, please exclude the edge case instead of disabling the rule globally